packages/ui-rating/src/Rating/props.ts

```diff
@@ -36,13 +36,39 @@ import type {
 } from '@instructure/shared-types'
 
 type RatingOwnProps = {
+  /**
+   * A label is required for accessibility
+   */
   label: string
+  /**
+   * A function that returns the current value formatted for screen readers
+   */
   formatValueText?: (...args: any[]) => any
```

Comment:
We know the type of this from the index file:

```suggestion
  /**
   * A function that returns the current value formatted for screen readers
   */
  formatValueText?: (filled: number, iconCount: number) => string
```